third-party/realdds/include/realdds/dds-exceptions.h

```diff
@@ -0,0 +1,38 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <librealsense2/utilities/easylogging/easyloggingpp.h>
+#include <stdexcept>
+#include <string>
+
+
+// E.g.:
+//     DDS_THROW( runtime_error, "what happened" );
+//
+#define DDS_THROW( ERR_TYPE, WHAT )                                                                 \
+    throw realdds::dds_ ## ERR_TYPE ( WHAT )
+
+
+namespace realdds {
+
+
+class dds_runtime_error : public std::runtime_error
```

Comment:
This will not really throw and only log it? or do both?

Reply:
throwing the class will throw :) this will just log it for now.